Source/ExecutionEngine/ExecutionEngine.cs

```diff
@@ -172,29 +173,21 @@ protected static string CleanUp(string msg)
 
   #endregion
 
-
-  public class ExecutionEngine : IDisposable
-  {
-    public static OutputPrinter printer;
-
-    public static ErrorInformationFactory errorInformationFactory = new ErrorInformationFactory();
+  public class ExecutionEngine : IDisposable {
+    public static ErrorInformationFactory ErrorInformationFactory { get; } = new();
 
     static int autoRequestIdCount;
 
     static readonly string AutoRequestIdPrefix = "auto_request_id_";
 
-    public static string FreshRequestId()
-    {
+    public static string FreshRequestId() {
       var id = Interlocked.Increment(ref autoRequestIdCount);
       return AutoRequestIdPrefix + id;
     }
 
-    public static int AutoRequestId(string id)
-    {
-      if (id.StartsWith(AutoRequestIdPrefix))
-      {
-        if (int.TryParse(id.Substring(AutoRequestIdPrefix.Length), out var result))
-        {
+    public static int AutoRequestId(string id) {
+      if (id.StartsWith(AutoRequestIdPrefix)) {
+        if (int.TryParse(id.Substring(AutoRequestIdPrefix.Length), out var result)) {
```

Comment:
Some of the changes above move curly braces on their own lines; this change does the reverse.  Maybe we should settle on one and apply it consistently?

Reply:
I think we're putting the braces on the same line, at least that what my Rider does, I assume based on implicit defaults on the `.editorconfig`. I see one place where this PR moves a brace to a new line, not sure how that happened.